forms/mainwindow.ui

```diff
@@ -355,7 +355,7 @@ border-bottom: 1px solid darkgrey;
                   </sizepolicy>
                  </property>
                  <property name="currentIndex">
-                  <number>0</number>
+                  <number>7</number>
```

Comment:
Try and revert to the default values for stacks and indexes before committing, so I don't have to do it after the merge.